node/src/service.rs

```diff
@@ -73,10 +73,16 @@ use {
     std::{pin::Pin, sync::Arc, time::Duration},
     substrate_prometheus_endpoint::Registry,
     tc_consensus::{
-        collators::basic::{self as basic_tanssi_aura, Params as BasicTanssiAuraParams},
+        collators::lookahead::{
+            self as lookahead_tanssi_aura, Params as LookaheadTanssiAuraParams,
+        },
+        //collators::basic::{self as basic_tanssi_aura, Params as BasicTanssiAuraParams},
         OrchestratorAuraWorkerAuxData,
     },
-    tokio::sync::mpsc::{unbounded_channel, UnboundedSender},
+    tokio::sync::{
+        mpsc::{unbounded_channel, UnboundedSender},
+        watch::Receiver,
```

Comment:
Can we use `watch::Receiver<()>` instead of `Receiver<()>` in this file? Because there may be other kinds of `Receiver`s

```suggestion
        watch,
```